pkg/utils/validation/features/featuregates.go

```diff
@@ -138,7 +138,7 @@ var featureGateVersionRanges = map[string]*FeatureGateVersionRange{
 	"IPTablesOwnershipCleanup":                         {Default: true, LockedToDefaultInVersion: "1.28", VersionRange: versionutils.VersionRange{RemovedInVersion: "1.30"}},
 	"ImageMaximumGCAge":                                {VersionRange: versionutils.VersionRange{AddedInVersion: "1.29"}},
 	"ImageVolume":                                      {VersionRange: versionutils.VersionRange{AddedInVersion: "1.31"}},
-	"InPlacePodVerticalScaling":                        {Default: true},
+	"InPlacePodVerticalScaling":                        {Default: true, LockedToDefaultInVersion: "1.33", VersionRange: versionutils.VersionRange{AddedInVersion: "1.27"}},
```

Comment:
How we decided that it is `LockedToDefaultInVersion: "1.33"`? I don't see the feature being locked to default neither in [release-v1.33](https://github.com/kubernetes/kubernetes/blob/f208b6c73d17f7c2535393117ab93cd18b673e47/pkg/features/kube_features.go#L1385-L1388) branch, nor in [master](https://github.com/kubernetes/kubernetes/blob/587aa60686c3bc4a0d43e29fe5f7fd27a5f3f5a7/pkg/features/kube_features.go#L1371-L1374) branch.

Reply:
Thanks for reviewing, had an offline discussion and it turns out I've misunderstood the `LockedToDefaultInVersion` utility and have used it incorrectly as the `InPlacePodVerticalScalling` is not yet locked and can be _enabled_/_disabled_ on demand.